### PR TITLE
Creating full bootstrap project for cloning NEWSWEEK site

### PR DESCRIPTION
**Pull request by rubenpazch**
This project has different requirements: 

+ Creating a responsive site using bootstrap 4
+ Top bar
+ Features articles 
+ top story
+ culture and travel
+ opinion
+ more stories
+ magazine section
+ featured slider
+ editors pick
+  subscribe and newsletter
+ footer

### CODE REVIEW

**Comment by majovanilla** [comment]
### Code review in progress by Maria Reyes

**Comment by majovanilla** [comment]
Hey @rubenpazch, 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯
I really liked the whole look of the page.

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Check your [HTML validations](https://validator.w3.org/nu/?doc=https%3A%2F%2Fgithub.com%2Frubenpazch%2FNewsweek-com-Bootstrap-%2Fblob%2Fdevelopment%2Findex.html). You have several errors and warnings.

- You need to use bootstrap classes **everywhere** I see that you are taking advantage of the grid classes, but your CSS files have some padding, margin, position and other common classes on bootstrap that you should be using on your HTML. I will point them on separate comments.
This is a Microverse requirement. See screenshot here:
Screenshot from 2020-02-21 17-38-12

- Your page doesn't behave exactly like the original one when you are on small screen sizes. The order of the sections should be: Top story, featured stories, opinion, more stories, culture and travel and latest news. Please make the necessary changes to match this order. I will give one screenshot as an example, but look for all the sections and make sure they are in order.

Your page:
Screenshot from 2020-02-21 17-46-51

Original:
Screenshot from 2020-02-21 17-47-01

- Check the More stories section on the small screen size, because it's not behaving like the original one. See screenshots:

Your page:
Screenshot from 2020-02-21 17-49-14

Original:
Screenshot from 2020-02-21 17-48-59

- The same with the opinion section on small screen sizes. See screenshots:

Your page:
Screenshot from 2020-02-21 17-51-27

Original:
Screenshot from 2020-02-21 17-51-33


- Please check the Latest news section also in small screen size.

Your page:
Screenshot from 2020-02-21 17-52-52

Original:
Screenshot from 2020-02-21 17-53-00

- In the magazine section behaves differently on small screens. The title should come after the green picture. Also, check the height of the green picture to fit the original version. 

Your page:
Screenshot from 2020-02-21 17-54-43

Original:
Screenshot from 2020-02-21 17-54-35

- The rest of the images on In the magazine section should have a rectangular proportion, yours is a square, please fix that too.

Your page:
Screenshot from 2020-02-21 17-56-42


Original:
Screenshot from 2020-02-21 17-56-28

- After the In the magazine section comes the featured slideshows and the the Editor's Pick. Your project is in a different order, please address that.

- From all the sections at the end, you have a different image size on the three small images with text. Please adjust them to match the original.

Your page:
Screenshot from 2020-02-21 18-00-07

Original:
Screenshot from 2020-02-21 18-00-00

- The last sections on medium screen sizes are also in the incorrect order and the side images have a different size. Please make them as the original page.

Your page:
Screenshot from 2020-02-21 18-02-31

Original:
Screenshot from 2020-02-21 18-02-16


- Use the hover styling on the footer as the original page.

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- You don't need to include all the ads on your page, especially when you are in md and sm sizes, because it doesn't look tidy. You could use a bootstrap class to hide them when reaching that screen width. See screenshot below:

YOUR PROJECT:
Screenshot from 2020-02-21 17-42-01

ORIGINAL:
Screenshot from 2020-02-21 17-42-52


## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes



You can find me on Social Media below 👇 . Remember to say hi!

[GitHub](https://github.com/majovanilla) [Twitter](https://twitter.com/MajoVanilla) [LinkedIn](https://www.linkedin.com/in/majoreyesparroquin/)

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,1188 @@
...
+  font-size: 20px;
+  font-family: var(--roboto-condensed);
+  color: var(--color-light-orange);
+  text-transform: inherit;
+  font-weight: bold;
+  letter-spacing: 1px;
+}
+
+.cascade-images {
+  position: relative;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for positioning.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,1188 @@
...
+  width: 50%;
+  height: 100%;
+  min-width: 275px;
+}
+
+.img-suscribe-1 {
+  position: absolute;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for positioning.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,1188 @@
...
+  left: 30px;
+  top: 25px;
+}
+
+.img-suscribe-2 {
+  position: absolute;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for positioning.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,1188 @@
...
+  top: 40px;
+  left: 86px;
+}
+
+.bg-header {
+  background-color: var(--color-light-orange);
+  color: var(--color-white);
+  height: 95px;
+  padding: 20px 10px 15px;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for padding.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,1188 @@
...
+}
+
+.bg-headerv2 {
+  background-color: var(--color-light-orange);
+  color: var(--color-white);
+  height: 50px;
+  padding: 11px 0 0 0;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for padding.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,1188 @@
...
+}
+
+.bg-header p {
+  margin-bottom: 0;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for margin.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,1188 @@
...
+}
+
+.bg-headerv2 p {
+  margin-bottom: 0;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for margin.

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,1188 @@
...
+}
+
+.logo {
+  text-align: center;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for text alignment.

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,1188 @@
...
+}
+
+.logo svg {
+  color: var(--color-white);
+  fill: var(--color-white);
+  width: 396px;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes for width.

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,1188 @@
...
+
+.logov2 svg {
+  color: var(--color-white);
+  fill: var(--color-white);
+  height: 20px;
+}
+
+.bg-suscribe {
+  text-align: right;
+  padding-right: 0;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes.

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,1188 @@
...
+.bg-suscribe a {
+  padding: 3px 10px;
+}
+
+.bg-suscribe svg {
+  width: 10px;
+  fill: var(--color-light-orange);
+  color: var(--color-light-orange);
+  padding: 0 0 5px 0;
+  margin: 0 0 0 14px;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes.

File: css/style.css [hunk 12]
```diff
@@ -0,0 +1,1188 @@
...
+.bg-suscribev2 a {
+  padding: 3px 10px;
+}
+
+.bg-suscribev2 svg {
+  width: 10px;
+  fill: var(--color-light-orange);
+  color: var(--color-light-orange);
+  padding: 0 0 5px 0;
+  margin: 0 0 0 14px;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes.

File: css/style.css [hunk 13]
```diff
@@ -0,0 +1,1188 @@
...
+  padding-top: 15px;
+  width: 86%;
+  padding-left: 0;
+}
+
+.lanscape-menu li {
+  display: inline-block;
+  padding-right: 0;
+  padding-left: 0;
+  text-align: center;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes.

File: css/style.css [hunk 14]
```diff
@@ -0,0 +1,1188 @@
...
+}
+
+.bg-headerv2 a {
+  text-decoration: none;
+}
+
+.featured-stories {
+  padding: 0 0 0 0;
+  width: 100%;
+  padding-top: 15px;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes.

File: css/style.css [hunk 15]
```diff
@@ -0,0 +1,1188 @@
...
+}
+
+.label-article-link {
+  position: absolute;
+  bottom: -220px;
+  left: 0;
+  line-height: 2rem;
+  text-align: center;
+  background-color: rgba(255, 255, 255, 0.9);
+  padding: 0 10px 0 10px;
```
Comment (majovanilla) [inline css/style.css]: Favor the use of bootstrap classes.

**Comment by majovanilla** [comment]
I left some comments on your code, but there are several classes that you could avoid using while favoring the use of the framework.

**Comment by rubenpazch** [comment]
thank you for your suggestions, I think I fix it so I'm sending it another time for review

**Comment by rubenpazch** [comment]
thank you for your suggestions, I think I fix it so I'm sending it another time for review

File: css/style.css (hunk 15 above)
Comment (rubenpazch) [reply css/style.css]: thank you

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review By Tolulope Olusakin(TSE)
Please check on this in a few minutes for some feedback :+1:

**Comment by nikasulo** [comment]
# Great work making the required changes! @rubenpazch 
You have done really well, however, a new problem has been introduced and your page is scrolling horizontally :frowning: Please look into this and make the adequate corrections! :handshake: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by rubenpazch** [comment]
Thank you for your comments, please can you send me a screenshot.

**Review by mosesogwo — APPROVED** [review]
**Great! Your project is approved:heavy_check_mark:**

You can now merge to master.

**Cheers:stars:**

### Status: Approved!

**Pull request merged**